index.js

```diff
@@ -62,4 +62,8 @@ module.exports = function(app) {
     app.route('/*').get(function(req, res) {
         res.sendFile(viewsDirectory + '/front.html');
     });
+
+    app.use(function (err, req, res) {
+      if (err) res.status(err.code || 500).send(err);
```

Comment:
Don't use brackets if you want, but please insert a newline between the condition and the body block if you don't `return`.